requirements.txt.lite

```diff
@@ -15,3 +15,4 @@ j2cli[yaml]
 j2cli
 lightrdf
 sssom
+babelon
```

Comment:
On second thought, not sure this belongs to ODKLite.

The rationale for ODKLite is that it should be the minimal ODK image needed to run the standard ODK workflows. As long as Babelon is not used in any of said workflows, I’d say it belongs to ODKFull only.

Reply:
Transcribing right now the "$(ONT)-international.owl" pipeline from HPO and MP which will allow people to add an internationalised release. Testing this right now, will make PR shortly.

Reply:
You mean that the HPO and MP internationalisation rules will get upstreamed into the ODK to become part of the standard workflows?

OK then, in that case no objection.